android/ironoxide-android/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Do we need this file. Nothing in it...

Reply:
I opened https://github.com/IronCoreLabs/ironoxide-java/issues/108 to figure this out